### PR TITLE
S5t start timer

### PR DESCRIPTION
**Pull request by PaulREnglish**
I hope my Paul Request gets approved.

### CODE REVIEW

File: functions.js [hunk 1]
```diff
@@ -159,12 +159,26 @@ const partyMode = () => {
     }, 3000)
 }
 
+/**
+ * function to start the timer
+ *
+ * @param {object} score the timer object that will have the timer and interval set
+ * @return {object} the timerobject returned with the timer and the interval set
```
Comment (emmatrain) [inline functions.js]: space between timer and object

File: index.js [hunk 2]
```diff
@@ -14,6 +13,10 @@ document.getElementById('startButton').addEventListener('click', () => {
     if (countries.length) {
         startGame()
         country = getAndDisplayCountry(countries)
+        timerObject = startTimer(timerObject)
+        setInterval(()=>{
+            console.log(timerObject.time)
```
Comment (emmatrain) [inline index.js]: should this be removed, was it just for testing?

File: functions.js (hunk 1 above)
Comment (emmatrain) [reply functions.js]: timerObject object

File: index.js (hunk 2 above)
Comment (PaulREnglish) [reply index.js]: yh it should be removed

**Pull request merged**